admin_manual/configuration_database/linux_database_configuration.rst

```diff
@@ -41,6 +41,57 @@ Configuring a MySQL or MariaDB database
 
 If you decide to use a MySQL or MariaDB database, ensure the following:
 
+* The transaction isolation level is set to "READ-COMMITED" in your MariaDB server configuration :file:`/etc/mysql/my.cnf` to persist even after a restart of your database server.
+
+  Verify the **transaction_isolation** and **binlog_format**:
+
+::
+
+  [mysqld]
+  ...
+  transaction_isolation = **READ-COMMITTED**
+  binlog_format = **ROW**
```

Comment:
Those `**` are not translated to bold text so I would drop it for now to not cause broken config. Same goes for the ones below. Beside that it looks good 👍

Reply:
Great! Thank you!

Reply:
Could you do those changes?

Reply:
done.